### ISSUE TITLE
OD radio - Datestamp A11y screen-reader requirements

### ISSUE THREAD

**Issue by RichardPK** [bbc/simorgh#6425]
**Completion of issue is pending A11y swarm 18/5/20**

**Is your feature request related to a problem? Please describe.**
As many OD radio episodes have the same title and description - the Datestamp becomes an important distinguisher between episodes. This means it ought to be read at the same time as the Episode title by screen readers.

Content order for screen readers:
The h1 should be announced first (Brand title & datestamp)
And then the image is announced with the alt-text
And that should be followed by the paragraph text,
And then the audio player

**Describe the solution you'd like**

- The Aria role "text" will enable the reading of the two elements continuously, but it will also disregard the semantics of any child html elements.
_i.e. this is only a half-solution. The text will be read out continuously, however it will not have any heading/p semantics:_

```
> <span role="text">
>   <h1>Magazine de la culture</h1>
>   <p>5 avril 2020</p>
> </span>
```

- In order to preserve the semantics of the html - the screen reader must be aware that this content is within a heading element. 
_i.e. it should all be wrapped within an `<h1>`, something like:_

```
> <h1>
>   <span role="text">
>     <span>Magazine de la culture</span>
>     <span>5 avril 2020</span>
>   </span>
> </h1>

```

- The outer `<h1>` ought to have the attributes `id=“content” tabindex=“-1"`

- The current 'HeadingBlock' component could be renamed BrandTitle, given that the 'Heading' will now encapsulate both the BrandTitle and the Datestamp

- Also consider, from @greenc05 "we should also ensure this renders correctly with no CSS e.g. that there is a space between ‘Magazine de la culture’ and ‘5 avril 2020’"
 
**Additional notes from Catharine are below:**

"With regards to HTML semantics, usually a datestamp would be a time HTML element. When I tested using a time HTML element within the h3/a for a promo this caused some issues with AT. Happy for the devs to use the TIME element here if they can conduct through testing of this within a h1, otherwise I would recommend the following

```
> <h1>
>   <span role="text">
>     <span>Magazine de la culture</span>
>     <span class="vh">, </span>
>     <span>5 avril 2020</span>
>   </span>
> </h1>
```

I believe role text will be required to stop voiceover splitting this into 2 elements"

**Notes on the HTML Time element**

Documentation of the Time element bug can be found here: https://docs.google.com/document/d/1Y6fkoXoAP0BLIHJfAD48MhnDil1XQ_QFSwNI5DN8ZPg/edit

Note that the bug was when the time element was used for duration - it might be worth trying it out to see if it is the same for the presentation title.

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**
